src/unistd.rs

```diff
@@ -3569,13 +3568,29 @@ impl Group {
     ///
     /// # Examples
     ///
+    // Disable this test on all OS except BSD as wheel group may not exist.
+    #[cfg_attr(not(any(apple_targets, bsd)), doc = " ```no_run")]
```

Comment:
`bsd` actually contains `apple_targets`, see [this](https://github.com/nix-rust/nix/blob/0dfcf3228489a5060a53e69783a1eea6f43b11a9/build.rs#L19).

Reply:
done